src/util/ens.ts

```diff
@@ -0,0 +1,42 @@
+import { keccak256 } from "@ethersproject/keccak256"
+import { EthNetworkID, IGatewayWeb3Client, VocdoniEnvironment } from "../common";
+import {
+    VOCDONI_GOERLI_ENTITY_ID,
+    VOCDONI_MAINNET_ENTITY_ID,
+    VOCDONI_RINKEBY_ENTITY_ID, VOCDONI_SOKOL_ENTITY_ID,
+    VOCDONI_XDAI_ENTITY_ID, VOCDONI_XDAI_STG_ENTITY_ID
+} from "../constants"
+import { IEnsPublicResolverContract } from "../net/contracts";
+
+export function getEnsPublicResolverByNetwork(
```

Comment:
As mentioned above, we could make this more generic and simply return the result of fetching the value on the contract